collectors/cochrane/parser.py

```diff
@@ -0,0 +1,83 @@
+except ImportError:
+    import xml.etree.ElementTree as etree
+from .record import Record
+
+
+def parse_record(url, file_path):
+    tree = etree.parse(file_path)
+    study_robs = []
+    studies = []
+
+    # Get risk of bias
+
+    root = tree.getroot()
+    doi_id = root.attrib.get('DOI', '')
+    quality_item_data_entries = tree.findall('//QUALITY_ITEM_DATA_ENTRY')
+    for quality_item_data_entry in quality_item_data_entries:
+        study_rob = {}
+        study_rob['study_id'] = quality_item_data_entry.attrib['STUDY_ID']
+        study_rob['modified'] = quality_item_data_entry.attrib.get('MODIFIED', '')
+        study_rob['result'] = quality_item_data_entry.attrib['RESULT']
+        study_rob['group_id'] = quality_item_data_entry.attrib.get('GROUP_ID', '')
+        study_rob['group_name'] = ''
+        for description in quality_item_data_entry.iter('P'):
+            study_rob['result_description'] = description.text
+
+        quality_item = quality_item_data_entry.getparent().getparent()
+        study_rob['rob_id'] = quality_item.attrib['ID']
+        study_rob['rob_name'] = quality_item.findtext('NAME')
+        rob_description = quality_item.find('DESCRIPTION/P')
+        study_rob['rob_description'] = rob_description.text
+        for group in quality_item.iter('QUALITY_ITEM_DATA_ENTRY_GROUP'):
+            group_id = group.attrib.get('ID')
+            if group_id == study_rob['group_id']:
+                study_rob['group_name'] = group.findtext('NAME')
+        study_robs.append(study_rob)
+
+    # Get references
+
+    included_studies = tree.find('//INCLUDED_STUDIES')
+    for study in included_studies.iter('STUDY'):
+        study_info = {}
+        study_info['doi_id'] = doi_id
+        study_info['file_name'] = file_path
+        study_info['id'] = study.attrib['ID']
+        corresponding_robs = [rob for rob in study_robs
+                              if rob['study_id'] == study_info['id']]
+        study_info['robs'] = corresponding_robs
+        study_info['study_type'] = study.attrib['DATA_SOURCE']
+        study_info['refs'] = []
+        for reference in study.iter('REFERENCE'):
+            ref = {}
+            ref['type'] = reference.attrib['TYPE']
+            ref['authors'] = reference.findtext('AU') or ''
+            ref['title'] = reference.findtext('TI') or ''
+            ref['source'] = reference.findtext('SO') or ''
+            ref['year'] = reference.findtext('YR') or ''
+            ref['vl'] = reference.findtext('VL') or ''
+            ref['no'] = reference.findtext('NO') or ''
+            ref['pg'] = reference.findtext('PG') or ''
+            ref['country'] = reference.findtext('CY') or ''
+            ref['identifiers'] = []
+            for identifier in reference.iter('IDENTIFIER'):
+                ident = deepcopy(identifier.attrib)
+                ident = {key.lower(): ident[key] for key in ident
+                         if key not in ['MODIFIED', 'MODIFIED_BY']}
```

Comment:
Fair enough, that's the right call then. Just change the way you generate the `ident` dict to avoid the need of a `deepcopy()` (as per my previous comment)